game_function.php

```diff
@@ -1,34 +1,66 @@
-
 <?php
-
+/**
+* @param object $db
+* @return array of games
+*/
+function getDB(): PDO
+{
+    $db = new PDO('mysql:host=db;dbname=boardStupid', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    return $db;
+}
 
 /**
- * @param array $result
- * @return string
- * loops through all data from boardStupid db and presents it
+ * @param $db
+ * @return mixed
  */
-function printStats(array $result): string
+function getBoardStupid($db)
 {
+    $query = $db->prepare('SELECT *  FROM `boardStupid`;');
+    $query->execute();
+    $result = $query->fetchAll();
+    return $result;
+}
 
-    $game_data = '';
-if (empty($result)){
+/**
+* @param array $result
+* @return string
+* loops through all data from boardStupid db and presents it
+*/
+function printStats(array $result): string
+{
+$game_data = '';
+if (empty($result)) {
     return 'There are no games to display';
-} else {
-    foreach ($result as $data) {
-        $game_data .='<div class="game">';
+}
+foreach ($result as $data) {
+    if (!$data['deleted']) {
```

Comment:
Ash told me to change this if statement and just check in your getBoardgames selector, where deleted = 0 if that makes sense? then you don't need this if statment.